qutebrowser/browser/greasemonkey.py

```diff
@@ -116,6 +120,13 @@ def _meta_json(self):
             'run-at': self.run_at,
         })
 
+    def add_required_script(self, source):
+        """Add the source of a required script to this script."""
+        # NOTE: If source also contains a greasemonkey metadata block then
+        # QWebengineScript will parse that instead of the actual one.
+        # Adding an indent to source would stop that.
```

Comment:
Hmm, so is there a reason to not add that indent (via `textwrap.indent` for example)?

Reply:
No, I guess I thought it would be too much of a pain for little gain. But If we can just throw a library function at it I guess I will. Hopefully it doesn't cause any issues.